config_defaults_inc.php

```diff
@@ -5117,3 +5119,15 @@
  * the attachment is linked to the note.  Or 0 for disabling this feature.
  */
 $g_issue_activity_note_attachments_seconds_threshold = 3;
+
+#################################
+# Simple Report Screen settings #
+#################################
+
+/**
+ * Define the access levels for which the simple entry screen should be presented
+ * Define the fileds to be displayed ( initial value is same as current set
+ * example : $g_simple_bug_report_page_fields = array(	'additional_info',	'attachments',	'category_id',);
+ */
+$g_simple_report_level = array( REPORTER, UPDATER, );
```

Comment:
Can be set to `$g_simple_report_level = UPDATER;` after having addressed my comment concerning usage of `access_has_project_level`

Reply:
The options should be moved in file, so that they are near to `$g_bug_report_page_fields`

Reply:
There is no PHPDoc comment for `$g_simple_report_level` and `$g_simple_bug_report_page_fields`

Reply:
The moving I understand, placed them initially at the bottom for easy finding, will address this in next commit.
As for the PHPDoc comments, I updated 2 XML files (fields & misc) within last commit, what other file needs to be updated?

Reply:
See here what PHPDoc is used for https://docs.phpdoc.org/3.0/guide/references/phpdoc/index.html

In this case, use the `@global` tag for the new variables
For example have a look how it is used  for `$g_bug_report_page_fields`